driven/storage/adapter.go

```diff
@@ -1315,6 +1372,238 @@ func (sa *Adapter) DeleteQueueDataForRecipientsWithContext(ctx context.Context,
 	return nil
 }
 
+// StoreAirshipToken stores airship token
+func (sa Adapter) StoreAirshipToken(orgID string, appID string, tokenInfo *model.TokenInfo, userID string) error {
```

Comment:
Since this function is very similar to `StoreFirebaseToken`, I think you can combine them into one `StoreToken` function that passes `tokenInfo.TokenType` to the helper storage functions so that they know which list of tokens to modify.